packages/main/src/SliderBase.ts

```diff
@@ -726,7 +758,23 @@ abstract class SliderBase extends UI5Element {
 	}
 
 	get _ariaLabelledByHandleRefs() {
```

Comment:
aren't these predefined? do we need getters? can we just hardcode them in the template?